### PR TITLE
redbiom.util.ids_from exact True->False

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

File: qiita_pet/handlers/qiita_redbiom.py [hunk 1]
```diff
@@ -58,7 +59,7 @@ def _redbiom_feature_search(self, query, contexts):
         study_artifacts = defaultdict(lambda: defaultdict(list))
         query = [f for f in query.split(' ')]
         for ctx in contexts:
-            for idx in redbiom.util.ids_from(query, True, 'feature', ctx):
+            for idx in redbiom.util.ids_from(query, False, 'feature', ctx):
```
Comment (charles-cowart) [inline qiita_pet/handlers/qiita_redbiom.py]: This explicit switch from True to False is intentional right?

File: qiita_pet/handlers/qiita_redbiom.py [hunk 2]
```diff
@@ -71,7 +72,9 @@ def _redbiom_taxon_search(self, query, contexts):
             # find the features with those taxonomies and then search
             # those features in the samples
             features = redbiom.fetch.taxon_descendents(ctx, query)
-            for idx in redbiom.util.ids_from(features, True, 'feature', ctx):
+            if len(features) > 600:
```
Comment (charles-cowart) [inline qiita_pet/handlers/qiita_redbiom.py]: Please add comment description for this magic number.

File: qiita_pet/support_files/doc/source/redbiom.rst [hunk 3]
```diff
@@ -15,22 +15,29 @@ Search Options
...
 
-    * Find all samples in which the word infant exists, as well as antibiotics, where the infants are under a year old:
+    N.B., the specific category in which a stem is found is not assured to be the same, *antibiot* could be in one category and *cry* in another. A set intersection can be performed with "&", a union with "|" and a difference with "-".
+  * **Value search:** In addition to the stem-based search, value based searches can also be a applied. These use a Python-like grammar and allow for a rich set of comparisons to be performed based on a metadata category of interest. For example, *where qiita_study_id == 10317* will find all samples which have the *qiita_study_id* metadata category, and in which the value for that sample is *10317*.
```
Comment (charles-cowart) [inline qiita_pet/support_files/doc/source/redbiom.rst]: a applied -> applied

File: qiita_pet/support_files/doc/source/redbiom.rst [hunk 4]
```diff
@@ -15,22 +15,29 @@ Search Options
...
+  * **Examples:**
+
+    * Find all samples in which both the word infant, as well as antibiotics exist, and where the infants are under a year old:
```
Comment (charles-cowart) [inline qiita_pet/support_files/doc/source/redbiom.rst]: 'infant' and 'antibiotics' might be better with single quotes or italics surrounding them.

File: qiita_pet/handlers/qiita_redbiom.py (hunk 1 above)
Comment (antgonza) [reply qiita_pet/handlers/qiita_redbiom.py]: This is the main reason we need this change!

File: qiita_pet/handlers/qiita_redbiom.py (hunk 2 above)
Comment (antgonza) [reply qiita_pet/handlers/qiita_redbiom.py]: done

File: qiita_pet/support_files/doc/source/redbiom.rst (hunk 3 above)
Comment (antgonza) [reply qiita_pet/support_files/doc/source/redbiom.rst]: thanks!

File: qiita_pet/support_files/doc/source/redbiom.rst (hunk 4 above)
Comment (antgonza) [reply qiita_pet/support_files/doc/source/redbiom.rst]: k

**Pull request merged**